topics/single-cell/tutorials/GO-enrichment/tutorial.md

```diff
@@ -0,0 +1,231 @@
+
+**[B] GO Enrichment Files:**
+
+We'll also use three additional files for GO enrichment analysis. 
+  * **Gene Ontology file:** This file contains information about Gene Ontology terms.
+  * **GO Annotations file:** This file maps genes to their corresponding GO terms.
+  * **Population set file:** This file provides a list of all genes involved in the experiment and is used as a background gene set for the analysis.
+
+*Note:* There are several online databases available for downloading GO and GO Annotations files, including the Gene Ontology website, Ensembl, and the UCSC Genome Browser.
+  
+>  <comment-title>Concept behind GO Enrichment Analysis</comment-title>
+>
+>  The goal of GO enrichment analysis is to interpret the biological significance of long lists of **marker genes**. By summarizing these genes into a shorter list of enriched **GO terms**. The analysis works by comparing each GO term between your list of marker genes and a **background gene set**. Statistical tests are then used to calculate a p-value that indicates whether a particular GO term is significantly enriched in the marker gene list compared to the background.
+{: .comment}
+
+# Get data
+
+You can access the data for this tutorial in multiple ways:
+
+1. **Importing from a history** - You can import [this history](https://usegalaxy.eu/u/mennagamal/h/go-enrichment-analysis-on-scrna-seq-data)
+
+   {% snippet faqs/galaxy/histories_import.md %}
+
+2. **Uploading from Zenodo** (see below)
+
+> <hands-on-title> Data Upload from Zenodo </hands-on-title>
+>
+> 1. Create a new history for this tutorial
+> 2. Import the files from [Zenodo]({{ page.zenodo_link }}) 
+>
+>    ```
+>    {{ page.zenodo_link }}/files/Galaxy3-[GO].obo
+>    {{ page.zenodo_link }}/files/Galaxy2-[GO_annotations_Mus_musculus].tabular
+>    {{ page.zenodo_link }}/files/Galaxy5-[Markers_-_clusters].tabular
+>    {{ page.zenodo_link }}/files/Galaxy4-[Background_gene_set].tabular
+>    {{ page.zenodo_link }}/files/Galaxy1-[Markers_-_genotype_].tabular
+>    ```
+>
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>
+>    {% snippet faqs/galaxy/datasets_import_from_data_library.md %}
+>
+> 3. Rename the datasets
+> 4. Check that the datatype is `tabular`
+>
+>    {% snippet faqs/galaxy/datasets_change_datatype.md datatype="tabular" %}
+>
+{: .hands_on}
+
+## Important tips for easier analysis
+
+{% snippet faqs/galaxy/tutorial_mode.md %}
+
+{% snippet topics/single-cell/faqs/single_cell_omics.md %}
+
+{% snippet faqs/galaxy/analysis_troubleshooting.md sc=true %}
+
+
+# Data processing
+
+To perform GO enrichment analysis on each cell cluster individually, we need to separate our "Markers_clusters Dataset" into seven files, one for each cluster and the "Markers_genotype Dataset" into 2 files, one for each condition. We'll use the "Split file" tool for this step. 
```

Comment:
It would be nice to clarify here what sort of enrichment analysis is the tool doing (ORA, GSEA, GSVA, etc) and the test used (usually hypergeometric for ORA for instance). This has some implications on the sensitivity/specificity of the results and the inputs that you would use.

Reply:
Great! I added that to the intro section and will add further explanation to the tutorial slides as well.